specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -13221,6 +16065,111 @@
         }
       }
     },
+    "FunctionEnvelope": {
+      "description": "Web Job Information.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "FunctionEnvelope resource specific properties",
+          "properties": {
+            "name": {
+              "description": "Function name.",
+              "type": "string"
+            },
+            "functionAppId": {
+              "description": "Function App ID.",
+              "type": "string"
+            },
+            "scriptRootPathHref": {
+              "description": "Script root path URI.",
+              "type": "string"
+            },
+            "scriptHref": {
+              "description": "Script URI.",
+              "type": "string"
+            },
+            "configHref": {
+              "description": "Config URI.",
+              "type": "string"
+            },
+            "secretsFileHref": {
+              "description": "Secrets file URI.",
+              "type": "string"
+            },
+            "href": {
+              "description": "Function URI.",
+              "type": "string"
+            },
+            "config": {
+              "$ref": "#/definitions/Object",
+              "description": "Config information."
+            },
+            "files": {
+              "description": "File list.",
+              "type": "object",
+              "additionalProperties": {
+                "type": "string"
+              }
+            },
+            "testData": {
```

Comment:
what needs to be provided as test data? better description will be helpful.

Reply:
Checked with owning dev; updated the description accordingly.  Fix should be in [Update #5](https://github.com/Azure/azure-rest-api-specs/pull/1551/commits/dfa48f604bd525511c285e45d39e09ac156e1ad8).